src/components/Flex/Flex.tsx

```diff
@@ -30,6 +33,10 @@ export type FlexAlignContent =
 export type FlexProps = {
     direction?: FlexDirection;
     wrap?: FlexWrap;
+    gap?: SpacingValue;
```

Comment:
@ryancarville While trying out with the control on the story, I'm a bit confused between gap and spacing. Gap doesn't seem to work at all.

Reply:
@imoutaharik good spot.  The `gap` prop was initially added to align the prop terminology with the css properties, but then I realised it would be a breaking change.  `spacing` is used for the `gap` css value and I will remove `gap` from the type definition.